relays/bin-substrate/src/cli/relay_headers_and_messages.rs

```diff
@@ -40,9 +40,9 @@ pub enum RelayHeadersAndMessages {
 /// Parameters that have the same names across all bridges.
 #[derive(StructOpt)]
 pub struct HeadersAndMessagesSharedParams {
-	/// Hex-encoded lane id that should be served by the relay. Defaults to `00000000`.
-	#[structopt(long, default_value = "00000000")]
```

Comment:
Could we not continue to have a default that's just `vec!["00000000"]`?

Reply:
Sorry - I was unable to interpret "not continue to have" part :) I guess you wanted to return default value for this param, right? I was going to remove it, because there's actually nothing special in lane 00000000 && imo it makes no sense to have default for this. But there are other commands where this default value is still used, so I've reverted it for now. If you mean something else, please respond here

Reply:
Haha, yeah giving it a second read it does sound confusing, sorry! But you got it, I wanted to keep the default value there.